src/cascadia/TerminalApp/MinMaxCloseControl.xaml

```diff
@@ -135,6 +135,7 @@ the MIT License. See LICENSE in the project root for license information. -->
             x:Name="MinimizeButton"
             x:Uid="WindowMinimizeButton"
             Style="{StaticResource CaptionButton}"
+            ToolTipService.ToolTip="Minimize"

```

Comment:
Should these strings be coming from our Resources so they can be localized?

Reply:
(this doesn't look like it was actually resolved)

Reply:
For the "maximize" button, how do one script it so that it displays "maximize" and "restore down"?

Reply:
> (this doesn't look like it was actually resolved)

wdym? It seems to be working.